src/EnergyPlus/SimulationManager.cc

```diff
@@ -1729,11 +1730,7 @@ namespace SimulationManager {
         // na
 
         // SUBROUTINE PARAMETER DEFINITIONS:
-        static ObjexxFCL::gio::Fmt EndOfDataFormat("(\"End of Data\")"); // Signifies the end of the data block in the output file
-        static std::string const ThreadingHeader("! <Program Control Information:Threads/Parallel Sims>, Threading Supported,Maximum Number of "
-                                                 "Threads, Env Set Threads (OMP_NUM_THREADS), EP Env Set Threads (EP_OMP_NUM_THREADS), IDF Set "
-                                                 "Threads, Number of Threads Used (Interior Radiant Exchange), Number Nominal Surfaces, Number "
-                                                 "Parallel Sims");
+        static constexpr auto EndOfDataFormat("End of Data\n"); // Signifies the end of the data block in the output file
```

Comment:
@lefticus so this `EndOfDataFormat` is used by the eio file, along with the eso and meter (time series) output files (down [here](https://github.com/NREL/EnergyPlus/pull/7746/files#diff-5b4ea1e38e96bd2ee33e0d0cf51d12b0R1882)).  Is there a reason you needed to put a newline here?  It looks like this is the culprit.